kafka-connect/kafka-connect/src/main/java/org/apache/iceberg/connect/data/RecordWrapper.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.connect.data;
+
+import java.util.Map;
+import org.apache.iceberg.data.Record;
+import org.apache.iceberg.types.Types.StructType;
+
+public class RecordWrapper implements Record {
```

Comment:
What is the purpose of this class? Why would you not just use the underlying `Record` directly?

Reply:
This was being used for CDC support, to carry the operation type with the record. But that's removed for now.